aeon/dj_pipeline/__init__.py

```diff
@@ -23,11 +27,11 @@ def get_schema_name(name) -> str:
 
 def dict_to_uuid(key) -> uuid.UUID:
     """Given a dictionary `key`, returns a hash string as UUID."""
-    hashed = hashlib.md5()
+    hashed = hashlib.sha256()
```

Comment:
Security: Replaced `hashlib.md5` with `hashlib.sha256` due to known vulnerabilities in the MD5 hashing algorithm.